dev/devicelab/bin/tasks/animated_complex_opacity_perf_impeller_ios__e2e_summary.dart

```diff
@@ -10,5 +10,5 @@ import 'package:flutter_devicelab/tasks/perf_tests.dart';
 
 Future<void> main() async {
   deviceOperatingSystem = DeviceOperatingSystem.ios;
-  await task(createAnimatedComplexOpacityPerfE2ETest(enableImpeller: true));
+  await task(createAnimatedComplexOpacityPerfE2ETest(testSkiaAndImpeller: true));
```

Comment:
I was going to only remove `complex_layout_scroll_perf_impeller_ios__timeline_summary` as an experiment, but maybe I should remove all of these (`animated_complex_opacity_perf_impeller_ios__e2e_summary` etc) to avoid increasing time but not reducing the number of builders. 
@zanderso wdyt?

Reply:
Sorry, not sure I understand. Do you mean removing all of the *_impeller_* variants, and only doing `testSkiaAndImpeller` everywhere? So not doing an experiment, and just converting everything in one fell swoop?

Reply:
The way I wrote this reused the `enableImpeller` parameter, changing the meaning from "only run impeller" to "run both Skia and impeller".  So if I land as-is then the current `impeller` benchmarks will run both, and their skia equivalent will only run the skia part, duplicating work.
So I can either
1. Accept that and remove the `impeller` benchmarks, and migrate the old Skia-only benchmarks to run both or
2. Refactor this to still have an `enableImpeller` that only runs Impeller, as well as a `testSkiaAndImpeller` capability that only `complex_layout_scroll_perf_ios__timeline_summary` uses.  And then if we like that pattern, later migrate the other Skia-only benchmarks to use `testSkiaAndImpeller` and remove the `impeller` benchmarks.